h/cli/commands/init.py

```diff
@@ -4,7 +4,7 @@
 import alembic.command
 import alembic.config
 import click
-import sqlalchemy
+from sqlalchemy.exc import ProgrammingError
```

Comment:
Make sure that `init.py` isn't doing anything with the real (non-patched) `sqlalchemy`: it only actually needs `ProgrammingError`